src/components/Notifications/Notifier.js

```diff
@@ -19,7 +19,7 @@ const Notifier = ({ notifications, removeSnackbar }) => {
     };
 
     React.useEffect(() => {
-        notifications.forEach(({ key, message, options = {}, dismissed = false }) => {
+        notifications.forEach(({ key, message, options = {}, contentOptions = {}, dismissed = false }) => {
```

Comment:
Add a comment to the addSnackbar function, mentioning the possible parameters, and especially the fact that `options` has precedence over `contentOptions`. If both are present and `options.content` is set, `contentOptions` will not be used